src/components/tag/readme.md

```diff
@@ -0,0 +1,16 @@
+```js
+const locationsValues = 5;
+const handleRemove = () => { console.info('Remove this tag'); };
```

Comment:
I find also useful log the param that you will receive as param, what do you think?